libs/langchain-google-gauth/src/tests/chat_models.int.test.ts

```diff
@@ -14,7 +14,7 @@ import {
 import { ChatGoogle } from "../chat_models.js";
 import { GoogleLLM } from "../llms.js";
 
-describe("GAuth Chat", () => {
+describe.skip("GAuth Chat", () => {
```

Comment:
Skipping because I copy/pasted these tests into `google-vertexai` since that's where people will actually be importing from